src/ComplexTypes/BaseLocation.php

```diff
@@ -21,7 +21,7 @@ class BaseLocation extends BaseType
     /**
      * @var string
      */
-    protected $OpeningTime = null;
+    protected $OpeningTime = '00:00:00';
```

Comment:
According to the documentation, `OpeningTime` is optional. But when you trigger `getNearestLocations` you will get an exception stating that null is not allowed. That's why i default to `00:00:00`.

Reply:
This is a confirmed bug on PostNL's side.